config/backstop/backstop.config.js

```diff
@@ -1,43 +1,44 @@
+let TEST_ENV_URL = 'http://localhost:3000';
+
+if (process.env.PR_ID) {
+  TEST_ENV_URL = `https://teaching-vacancies-review-pr-${process.env.PR_ID}.london.cloudapps.digital`;
+}
+
+let scenarios = [
+  ...require('./scenarios/public'),
+  ...require('./scenarios/common')
+];
+
+if (process.argv[4]) {
+  scenarios.push(...require(`./scenarios/${process.argv[4]}`));
+}
+
+scenarios = scenarios.map((scenario) => {
+  return {...scenario, url: `${TEST_ENV_URL}${scenario.url}`}
+})
+
 module.exports = {
   "id": "teacher_vacancies",
   "viewports": [
     {
       "label": "phone",
       "width": 320,
```

Comment:
I can't remember. Did we say we'd change this?

Reply:
we did but its not a task for now